### ISSUE TITLE
running on multiple package is not working

### ISSUE THREAD

**Issue by dvyukov** [golangci/golangci-lint#469]
```
$ golangci-lint --version
golangci-lint has version v1.15.0-21-gf516771 built from f516771 on Thu Mar 28 14:06:02 CET 2019
$ go version
go version go1.12 linux/amd64
```
Checkout https://github.com/google/syzkaller/commit/14c58f8d239741c39ddab01a4f0e95787cf99cc3 for code and .golangci.yml config.

Running `golangci-lint run ./...` does not produce any warnings even if I comment out `issues.exclude` section. Though, I know there are lots of these warnings. If I run `golangci-lint run ./pkg/osutil` with commented out `issues.exclude` it does produce a bunch of these warnings.

However, if I comments out `linters.disable` section and run `golangci-lint run ./...` it also produces some warnings. So it's kinda working to some degree. I can't understand this behavior and I can't trust it when it says I have no warnings, because I know it lies.

Now if i do:
```
   skip-files:
-    - "tools/syz-trace2syz/parser/(lex.go|strace.go|straceLex.rl|yaccpar)"
+    - "tools/syz-trace2syz/parser/(lex|strace).go"
```
and run `golangci-lint run ./...` I get a bunch of gocyclo warnings in completely unrelated files. These warnings should have been suppressed by //nolint comments. I am lost.

**Comment by jirfag** [comment]
Hi, thank you! I've made fixes https://github.com/golangci/golangci-lint/pull/504 and https://github.com/golangci/golangci-lint/pull/505. Please, check it now.

**Comment by dvyukov** [comment]
Now everything seems to work as expected. Thanks!
